sdcm/logcollector.py

```diff
@@ -1233,6 +1235,29 @@ def get_gce_instances_by_testid(self):
         if self.params["use_cloud_manager"]:
             self.find_and_append_cloud_manager_instance_to_collecting_nodes()
 
+    def create_collecting_nodes(self):
+        try:
+            provisioners = provisioner_factory.discover_provisioners(backend=self.backend, test_id=self.test_id)
+            instances = sum([provisioner.list_instances() for provisioner in provisioners], [])
+            collecting_nodes = [CollectingNode(name=instance.name,
+                                               ssh_login_info={
+                                                   "hostname": instance.public_ip_address,
```

Comment:
this function is in theory, would be the same one for aws/gce (once their provision would match), right ?

is so, please use public only if it's says so on the `param.get('ip_ssh_connections')`

Reply:
But we'll fix it once we'll get to it